lib/KCard/index.vue

```diff
@@ -205,10 +231,19 @@
           height: '100%',
         };
 
+        const titleAlignStyle = {
```

Comment:
I believe that `titleAlignStyle` can be removed completely after you utilize switching between `flex-end` and `flex-start`. Explained in mode detail in other comments.

Reply:
I think we need still need this to handle  switching between the title.

Reply:
Response here https://github.com/learningequality/kolibri-design-system/pull/706#discussion_r1718275985